zipkin-server/src/test/java/zipkin2/server/internal/ZipkinServerConfigurationTest.java

```diff
@@ -143,8 +105,19 @@ public void httpCollector_canDisable() {
       return new OrderedHealthAggregator();
     }
 
-    @Bean MeterRegistry registry () {
+    @Bean MeterRegistry registry() {
       return new PrometheusMeterRegistry(PrometheusConfig.DEFAULT);
     }
   }
+
+  static void registerBaseConfig(AnnotationConfigApplicationContext context) {
+    context.register(
+      ArmeriaSpringActuatorAutoConfiguration.class,
+      HealthStatusHttpMapper.class,
```

Comment:
this is the only needed change here `HealthStatusHttpMapper` is now a required bean